src/main/java/com/liveramp/captain/daemon/RequestUpdater.java

```diff
@@ -19,4 +19,8 @@ default void retry(long id) {
   default void quarantine(long id) {
 
   }
+
+  default void ready(long id) {
```

Comment:
What is this needed for? I think we should be very intentional when adding stuff to this class. This an open source library, and this is one of its core interfaces. From the other PR, it doesn't really look like why are using this for much

Reply:
If we mark the request as retry we dont need to do this.